package.json

```diff
@@ -632,6 +697,11 @@
                         "items": {
                             "type": "string"
                         }
+                    },
+                    "azureResourceGroups.enableChatStandIn": {
```

Comment:
So just to check, the `configurationDefaults` will override this?

Reply:
Yes, I've experimented and it seems to be working as intended.